src/Controller/SlackController.php

```diff
@@ -81,18 +80,13 @@ public function authenticate(Request $request, SlackApplication $slackApplicatio
                 'code' => $request->query->get('code'),
             ]);
 
-            // Who Am I?
-            /** @var SlackResourceOwner $user */
-            $user = $provider->getResourceOwner($token);
+            $admin = $userExtractor->getUser($token->getToken(), $token->getValues()['authed_user']['id']);
```

Comment:
Looks good to me except for this piece of code, deep nested array are scary! :jack_o_lantern: 

Let's ship it!